Source/InitEB.cpp

```diff
@@ -365,8 +366,12 @@ initialize_EB2(
 
   // Add finer level, might be inconsistent with the coarser level created
   // above.
-  if (geom_type != "chkfile") {
+  // EY: This condition is not acceptable in AMReX with stl format
```

Comment:
Instead of a silent comment, maybe we could replace this with an actual Abort? That way we know it breaks loudly and we can always go back and check if amrex accepts it later? Or maybe they never will? Or maybe I have no idea what I am talking about.

Reply:
The invalid condition gets caught by the AMREX_ALWAYS_ASSERT below. The addFineLevels function isn't supported for chkfile and stl, but it's ok to just skip if no levels are being added. So we catch the error for either of these options only when `max_level != eb_max_level`